app/build.gradle

```diff
@@ -123,17 +123,15 @@ custom.eachFile() { file ->
     return
   }
   map.put(fileName, file.getAbsolutePath())
-
-
 }
 
 android {
-  compileSdkVersion 24
-  buildToolsVersion '25.0.0'
+  compileSdkVersion 25
```

Comment:
Why switching to sdk version 25 ?

Just to get the last version or there is something the sdk 25 fixes ?
(And I would have prefer to have this change in a separate PR than hidden in the lots of unrelated commits. Especially as it impacts kiwix-build)